client/components/share/facebook-share-preview/style.scss

```diff
@@ -0,0 +1,73 @@
+.facebook-share-preview {
+	background-color: $white;
+	border: 1px solid;
+	border-color: #e5e6e9 #dfe0e4 #d0d1d5;
+	border-radius: 3px;
+	color: #1d2129;
+	margin: 0 auto;
+	max-width: 486px;
+	padding: 12px;
+	-webkit-overflow-scrolling: touch;
+}
+
+.facebook-share-preview a {
```

Comment:
@iamtakashi let's try and follow our css guidelines. In this case, we can add a classname like `facebook-share-preview__link` to each a tag, so we're able to use a selector like: `.facebook-share-preview__link` https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#csssass-coding-guidelines

@lamosty do you mind updating the markup and selectors?